### DISCUSSION TITLE
Easy binary download through GitHub Releases

### DISCUSSION THREAD

**Discussion by ghost** [pulsar-edit/.github#25] [General]
Using GitHub Releases, I have created a repository that makes it simple to browse through binaries!

https://github.com/softcode589/pulsar-binaries-gh-releases/releases

**Comment by kaosine** [comment]
I was actually about to handle this with badges and such on the main repo and buttons on the website. Just have gotten to it. I have a note that I will be using for doing this to make it easier hopefully when I get back from dinner.

**Reply by kaosine** [reply]
It also doesn't help that right now, what we're getting and building from there isn't exactly ready for releases and needs to be tagged and reversioned so that package managers(esp GUI ones like Discover) don't glitch out on versioning. Since instead of displaying that there is an update, it acts like it's going to remove it and not sure why. This is even one of the points we've got to establish for that or anything else to make sure clear links, and everything else are posted and even use releases. But that routes back to the whole tagging because we don't need people seeing these as stable...
